### ISSUE TITLE
Add more examples

### ISSUE THREAD

**Issue by bbarao** [bbarao/seguranca-sqli-xss#1]
We need more examples of good and bad code.

Current examples in other languages would also be nice.

**Comment by pizzot** [comment]
I suggest you start with adding a README.md file that would explain what the repository is about. Otherwise it is hard for someone to contribute even if he wanted to.

**Comment by bbarao** [comment]
@okneloper Yeah, you are right. Added a simple README for now, should help to explain the purpose.

**Comment by pizzot** [comment]
Great, thanks :)
